docs/styleguide-wording.md

```diff
@@ -0,0 +1,35 @@
+# Styleguide: Wording & Phrasing
```

Comment:
I don't know. This styleguide can and should be a living document -- if we want to decide on a preference, then we can make a future pull request to modify this document!